### PR TITLE
Added Getting Started to SDKs that aren't unified

### PR DESCRIPTION
**Pull request by MimiDumpling**
I've added Getting Started 1, 2, 3 to the index of each SDK in the main navigation that isn't unified. This will help users who miss the Getting Started under Error Reporting.

### CODE REVIEW

File: src/collections/_documentation/clients/cocoa/index.md [hunk 1]
```diff
@@ -6,8 +6,15 @@ sidebar_relocation: platforms
 
 This is the documentation for our official clients for Cocoa (Swift and Objective-C). Starting with version `3.0.0` we’ve switched our internal code from Swift to Objective-C to maximize compatibility. Also we trimmed the public API of our sdk to a minimum. Some of the lesser used features that were present before are gone now, check out [Migration Guide]({%- link _documentation/clients/cocoa/migration.md -%}#migration) or [Advanced Usage]({%- link _documentation/clients/cocoa/advanced.md -%}#advanced) for details.
 
+## Getting Started
+Getting started with Sentry is a three step process:
+
+1.  [Sign up for an account](https://sentry.io/signup/)
+2.  [Install your SDK](#install)
+2.  [Configure it](#configure)
```
Comment (lobsterkatie) [inline src/collections/_documentation/clients/cocoa/index.md]: You seem to have 1, 2, 2 for most of these steps...

File: src/collections/_documentation/clients/java/index.md [hunk 2]
```diff
@@ -4,27 +4,82 @@ sidebar_order: 6
 sidebar_relocation: platforms
 ---
 
-Sentry for Java is the official Java SDK for Sentry. At its core it provides a raw client for sending events to Sentry, but it is highly recommended that you use one of the included library or framework integrations listed below if at all possible.
+Sentry for Java is a collection of modules provided by Sentry. To begin, we **highly recommend** you use one of the library or framework integrations listed under Installation. Otherwise, [manual usage]({%- link _documentation/clients/java/usage.md -%}) is another option.
```
Comment (lobsterkatie) [inline src/collections/_documentation/clients/java/index.md]: Curious about why this change - the original seems slightly more accurate.

File: src/collections/_documentation/clients/java/index.md [hunk 3]
```diff
@@ -4,27 +4,82 @@ sidebar_order: 6
...
+-   [Google App Engine]({%- link _documentation/clients/java/modules/appengine.md -%})
+-   [java.util.logging]({%- link _documentation/clients/java/modules/jul.md -%})
+-   [Log4j 1.x]({%- link _documentation/clients/java/modules/log4j.md -%})
+-   [Log4j 2.x]({%- link _documentation/clients/java/modules/log4j2.md -%})
+-   [Logback]({%- link _documentation/clients/java/modules/logback.md -%})
+-   [Spring]({%- link _documentation/clients/java/modules/spring.md -%})
+
+## Configuration {#config}
+
+Use the configuration below in combination with any of the integrations from above. The configuration will only work after an integration is installed. After that, [set you DSN]{#setting-the-dsn}.
```
Comment (lobsterkatie) [inline src/collections/_documentation/clients/java/index.md]: "set you DSN"

**Comment by MimiDumpling** [comment]
@lobsterkatie, made the changes you suggested. The only one that might need review is the intro for the Java index.
